docs/guides/plugin_guide.rst

```diff
@@ -25,6 +18,13 @@ Data providers such as ArcGIS_, Geoserver_, and others are implemented as plugin
 .. _Geoserver: https://github.com/ngageoint/opensphere/tree/master/src/plugin/ogc
 .. _os.data.IDataProvider: https://github.com/ngageoint/opensphere/blob/master/src/os/data/idataprovider.js
 
+.. toctree::
+  :maxdepth: 1
+  :caption: Step-by-step Guides
+
+  plugin_file_type_guide/index
+  plugin_server_guide/index
+
```

Comment:
I'm not sure I like this move. With the File Guide, we simply removed the quick-and-dirty section on it from this page. We should probably move this back and remove the "New Server/Provider Types" section.

Reply:
I'll back out that part, and have another attempt (probably bigger edit) in a separate PR.